src/test/mgr/CMakeLists.txt

```diff
@@ -1,16 +1,21 @@
 # unittest_mgr_mgrcap
 add_executable(unittest_mgr_mgrcap
-  test_mgrcap.cc
-  $<TARGET_OBJECTS:mgr_cap_obj>)
+	test_mgrcap.cc
```

Comment:
[tabs, arghh](https://www.youtube.com/watch?v=SsoOG6ZeyUI)!

Reply:
it looks like I have a command in my config doing this xD, I won't let you down again sensei.